src/App.js

```diff
@@ -60,26 +58,38 @@ class Form extends Component{
 
     render(){
         return(
-            <div>
-            <Row className = "head" id = "form">
-                <Col span={8}>
-                    <Title level={2}>Stay tuned!</Title>
-                    <Text >Suscribe to be up to date of our new courses</Text>
-                    <RoundButton
-                        buttonText='Submit'
-                        onClick={() => console.log('Submit button clicked')}
-                    />
-                </Col>
-            </Row>
-            <Row className = "body" id = "form">
-                <Col span={24}>
-                    <AutoComplete
-                    style={{ width: 200 }}
-                    placeholder="Email"
-                    />
-
-                </Col>
-            </Row>
+            <div style = {{margin: 50}}>
+                <Row>
+                    <Col xs={3} sm={3} md={2} lg={1} />
```

Comment:
From now on, use `span={3}` by default and use `md=` to change the value for screen sizes bigger than mobile